pkg/cloudprovider/types.go

```diff
@@ -58,7 +58,7 @@ type NodeGroup interface {
 // Capacity provisions a set of nodes that fulfill a set of constraints.
 type Capacity interface {
 	// Create a set of nodes to fulfill the desired capacity given constraints.
-	Create(context.Context, *CapacityConstraints) (CapacityPacking, error)
+	Create(context.Context, *CapacityConstraints) (Packings, error)
```

Comment:
thoughts on renaming CapacityConstraints to Constraints if we're dropping Capacity from Packings? I'd eventually like to deprecate the other cloud providers (NodeGroup/Queue), or at least separate them.